### ISSUE TITLE
keep proxy connection

### ISSUE THREAD

**Issue by geekboood** [aio-libs/aiohttp#4750]
🐞 **Describe the bug**

Hi, I try to use the proxy with aiohttp but encounter a problem, that aiohttp close the connection to the proxy every time it finished the post request through the proxy.  

💡 **To Reproduce**

The code I used look like this (only the key logic):
```conn = aiohttp.TCPConnector(ssl=False, limit=None)
async with aiohttp.ClientSession(loop=loop, connector=conn) as session:
    tasks = [asyncio.create_task(get(queue)) for _ in range(256)]
```
and the core of get function look like this:
`
async with session.post(url, proxy=proxy_url) as resp:
`
with the command `netstat -npt | grep $PID | wc -l`, I can always observe the result is less than or equal to 256. 

💡 **Expected behavior**

I want to keep the proxy connection. I have over 1000 IP in the proxy pool, so the result of command `netstat -npt | grep $PID | wc -l` could be over 1000.

**Comment by bitphage** [comment]
See https://github.com/aio-libs/aiohttp/pull/3070

Btw, I've recently got it too and it would be really nice to have a flag to enable/disable proxy keepaliving.

**Comment by bitphage** [comment]
Actually looks like proxy connections aren't closed because of bug in #3070, so you can reuse proxy connections on per host:port basis, just make sure that keepalive timeout is high enough:

```python
connector = aiohttp.TCPConnector(keepalive_timeout=120)
client = aiohttp.ClientSession(connector=connector)
```

**Comment by dzhalaevd** [comment]
Hi, @geekboood, there is no activity here for the last 3 years, let's close this issue for now and open it again if necessary.
